test/unit/app/renderer/components/download/downloadBarTest.js

```diff
@@ -92,11 +126,11 @@ describe('downloadsBar component', function () {
     })
   })
 
-  describe('very narrow downloads bar with items', function () {
+  // skipped until #9176 is merged
+  describe.skip('very narrow downloads bar with items', function () {
```

Comment:
@NejcZdovc would you please add that to https://github.com/brave/browser-laptop/issues/5389 for a reminder?